setup.py

```diff
@@ -18,7 +18,7 @@
     include_package_data=True,
     install_requires=[
         'eth-utils>=1.2.0,<2.0.0',
-        'eth-typing<2',
+        'eth-typing==2.0.0',
```

Comment:
We typically use version ranges for install requirements.  That makes it so that things are less likely to break when several packages are installed at once that all require `eth-typing`.
```suggestion
        'eth-typing>=2.0.0,<3.0.0',
```